### DISCUSSION TITLE
How to render fillPattern with base64 image in mapbox

### DISCUSSION THREAD

**Discussion by somasekharkakarla** [rnmapbox/maps#3047] [Q&A]
i am using styling currently as
```
fillStyles: {
      lineColor:
       groups.length > 0
          ? ['match', ['get', idType], ...groups]
          : '#000000',
    }
```

i want to know how can i use fillPattern using base64. Need an example

Thank you in Advance

**Comment by mfazekas** [comment]
fillPattern accepts a resolved image. No base64 supported there. You might be able to get base64 working via `Images` component, you can also try the @rnmapbopx Image component  with an rn Image with base64 picture.

**Comment by somasekharkakarla** [comment]
Hi @mfazekas ,
But in web base64 works, we are getting on both as url and base64. Can you support base64 for fill pattern. What is very helpful

**Comment by somasekharkakarla** [comment]
@mfazekas by supporting base64 we can use that in offline also
